contracts/reverseRegistrar/ReverseRegistrar.sol

```diff
@@ -86,7 +87,7 @@ contract ReverseRegistrar is Ownable, Controllable, IReverseRegistrar {
         address owner,
         address resolver
     ) public override authorised(addr) returns (bytes32) {
-        bytes32 labelHash = sha3HexAddress(addr);
+        bytes32 labelHash = LowLevelCallUtils.sha3HexAddress(addr);
```

Comment:
If you do `using LowLevelCallUtils for address;` you can then do `addr.sha3HexAddress()`, which is much cleaner.